web/source/kmwcallback.js

```diff
@@ -54,7 +54,7 @@
         Lelem._KeymanWebSelectionEnd=Le;
         keymanweb._IgnoreNextSelChange = 0;
         if(Ptext!=null)keymanweb.KO(0, Lelem, Ptext);
-        if(typeof(PdeadKey)!='undefined') keymanweb.KDO(0, Lelem, PdeadKey);
+        if((typeof(PdeadKey)!=='undefined') && (PdeadKey !== null)) keymanweb.KDO(0, Lelem, PdeadKey);
```

Comment:
The only one of those that matters is `KeymanWeb.KT(attributes['x'].value,0);  // european2-1.6.js`.  None of the others bothered defining a value for the argument `Pdeadkey`.

In regard to that one, `0` wouldn't have been considered `'undefined'` anyway, would it?  I don't see a functionality change there.